src/sfizz/MathHelpers.h

```diff
@@ -39,6 +39,18 @@ constexpr T min(T op1, Args... rest)
     return min(op1, min(rest...));
 }
 
+/**
+ * @brief Compute the square of the value
+ *
+ * @param op
+ * @return T
+ */
+template<class T, class... Args>
```

Comment:
Not sure you need the `class... Args` here. However you could make it generic I guess, like  https://godbolt.org/z/gBefk4

Reply:
Good catch yes, this was a copy&paste error